.gitmodules

```diff
@@ -3,7 +3,7 @@
   url = git://github.com/omapsapp/osmctools.git
 [submodule "tools/kothic"]
   path = tools/kothic
-  url = git://github.com/omapsapp/kothic.git
+	url = git://github.com/organicmaps/kothic
```

Comment:
Is it a tab?

Reply:
Yes, this is how `git submodule set-url` works.

Reply:
Hm, but the rest of file uses notation like git://github.com/omapsapp/osmctools.git

@biodranik Should we fix all or leave unchanged?

Reply:
I prefer the same delimiters everywhere (spaces). But it's not a blocker, just a nit.

Reply:
No, I mean that other entries use **omapsapp**/osmctools**.git**

Reply:
Guys, could you please update .gitmodules in separate PR to 
a) Use https:// instead of git://
b) Use organicmaps instead of maps

Reply:
It's better to fix it here.

Reply:
@Dushistov could you please use the same formatting for .gitmodules? Just to be consistent.